### PR TITLE
Add test jobs in openvino_model_server

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**

### CODE REVIEW

**Review by Xaenalt — APPROVED** [review]

File: ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml [hunk 1]
```diff
@@ -229,6 +235,37 @@ resources:
     requests:
       cpu: "6"
       memory: 12Gi
+tests:
+- as: unit
```
Comment (Xaenalt) [inline ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml:239]: OVMS doesn't need the make test part, can remove it, the mirroring is great though!

**Review by Xaenalt — CHANGES_REQUESTED** [review]

File: ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml (hunk 1 above)
Comment (vaibhavjainwiz) [reply ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml:239]: done

**Comment by vaibhavjainwiz** [comment]
/pj-rehearse pull-ci-red-hat-data-services-openvino_model_server-v2022.3-release-pr-image-mirror

**Comment by vaibhavjainwiz** [comment]
/pj-rehearse pull-ci-red-hat-data-services-openvino_model_server-v2022.3-release-pr-image-mirror

**Pull request closed without merging**